app/index.js

```diff
@@ -4,28 +4,27 @@
  * @copyright jQuery Foundation and other contributors, https://jquery.org/
  * MIT License
  */
-"use strict";
 
 //------------------------------------------------------------------------------
 // Requirements
 //------------------------------------------------------------------------------
 
-const Generator = require("yeoman-generator");
+import Generator from "yeoman-generator";
+import RuleGenerator from "../rule/index.js";
+import PluginGenerator from "../plugin/index.js";
+import { fileURLToPath } from "node:url"; // eslint-disable-line node/no-missing-import -- https://github.com/mysticatea/eslint-plugin-node/issues/275
```

Comment:
Just double checking that the “node:” prefix is supported in all of the Node.JS versions we support? (12, 14, 16)?

Reply:
Yep. Unfortunately, eslint-plugin-node hasn't released a new version with the fix yet.